packages/portal/src/app/shared/errors/not-found/not-found.component.scss

```diff
@@ -0,0 +1,68 @@
+@import url('https://fonts.googleapis.com/css2?family=Black+Ops+One&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Black+Ops+One&display=swap');
+
+h1 {
+  // font-family: 'Black Ops One', cursive;
+  font-family: 'Bebas Neue', cursive;
+}
+
+._404-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    text-align: center;
+
+    h1 {
+      font-size: 4.5em;
+      margin-top: .5em;
+    }
+
+    ._404-header {
+      font-size: 3em;
+      line-height: 1em;
+      padding: 2%;
+    }
+
+    ._404-child {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+
+        button {
+          margin-right: 10%;
+          padding: 10px;
+        }
+    }
+}
+
+img {
+    max-width: 700px;
+}
+
+@media
+only screen and (max-width: 992px) {
+    ._404-container {
```

Comment:
We are currently implementing BEM as the pattern and naming convention for classes across the project (although it's possible there are some non-compliant cases). Here is an example:
```

.container // (block)
.container__wrapper // (block + element)
.container__wrapper--404page // (block + element + modifier) 
```

Although the recommendation to keep styles more manageable and reduce maintenance efforts, is to try to keep styles as generic as possible. Since we want to stay consistent and repeat visual cues across the sites, there is probably a specific reason why the styles are applied, for example to give them accent. Then the class name would look like

`.container__wrapper--accent`

Now we only have to maintain that specific style in one place, and we can apply it to many different pages (404 and other pages, error pages and even content pages), for consistency.

Reply:
Pushed the latest changes, used text instead of an image.

- Here's a screenshot ![404-page](https://res.cloudinary.com/dv765kdgq/image/upload/v1692800970/404-text_e3kjyx.png)
- Mobile view ![404-page](https://res.cloudinary.com/dv765kdgq/image/upload/v1692800886/404-text-mobile_phm7ru.png)